mappings/net/minecraft/entity/ai/brain/sensor/HoglinSpecificSensor.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_4832 net/minecraft/entity/ai/brain/sensor/HoglinSpecificSensor
+	METHOD method_24641 findNearestVisibleWarpedFungi (Lnet/minecraft/class_3218;Lnet/minecraft/class_4760;)Ljava/util/Optional;
```

Comment:
I think it could be shortened to `findNearestWarpedFungi`. The fact that hoglins need to see the fungi (and can't, for example, smell it) is just an implementation detail which can go in a comment.

Reply:
Mojang call the sensor "nearest visible warped fungi", but your name is better.